### PR TITLE
Add Lizzard73 bio

### PR DESCRIPTION
**Pull request by Lizzard73**
This Pull Request addresses issue #12 

Can you comment @LostThought ?

Pretty please? with a cherry :cherries: on top?

### CODE REVIEW

**Comment by Lizzard73** [comment]
I forgot to put emojis in my file

File: docs/Lizzard73_bio.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#Lizzard73
+
+##Name: Liz
+##Occupation: Time Traveler
+##Location: Oz
```
Comment (smbogan) [inline docs/Lizzard73_bio.md]: Please specify where in Oz, for example: The Emerald City, Land of the Munchkins, the Wicked Witches Tower, etc...

File: docs/Lizzard73_bio.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+#Lizzard73
+
+##Name: Liz
+##Occupation: Time Traveler
+##Location: Oz, Land of Munchkins
+
+guess I can't add emojis to this file
```
Comment (Lizzard73) [inline docs/Lizzard73_bio.md]: remove line 7

File: docs/Lizzard73_bio.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+#Lizzard73
+
+##Name: Liz
+##Occupation: Time Traveler
```
Comment (ghost) [inline docs/Lizzard73_bio.md:4]: I dont believe this to be true.

File: docs/Lizzard73_bio.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
...
+##Location: Oz, Land of Munchkins
```
Comment (ghost) [inline docs/Lizzard73_bio.md:5]: do you have a better understanding of git now?

**Comment by LostThought** [comment]
Here is my comment you requested :smile:

File: docs/Lizzard73_bio.md (hunk 3 above)
Comment (Lizzard73) [inline docs/Lizzard73_bio.md:4]: I'm pretty short ...

**Pull request merged**